connectors/sources/s3.py

```diff
@@ -231,33 +232,53 @@ def get_default_configuration(cls):
         """
         return {
             "buckets": {
-                "value": "ent-search-ingest-dev",
+                "display": "textarea",
                 "label": "AWS Buckets",
+                "order": 1,
                 "type": "list",
+                "value": "ent-search-ingest-dev",
             },
             "read_timeout": {
-                "value": 90,
+                "default_value": DEFAULT_READ_TIMEOUT,
+                "display": "numeric",
                 "label": "Read timeout",
+                "order": 2,
+                "required": False,
                 "type": "int",
+                "value": DEFAULT_READ_TIMEOUT,
             },
             "connect_timeout": {
-                "value": 90,
+                "default_value": DEFAULT_CONNECTION_TIMEOUT,
+                "display": "numeric",
                 "label": "Connection timeout",
+                "order": 3,
+                "required": False,
                 "type": "int",
+                "value": DEFAULT_CONNECTION_TIMEOUT,
             },
             "max_attempts": {
-                "value": 5,
+                "default_value": DEFAULT_MAX_RETRY_ATTEMPS,
+                "display": "numeric",
                 "label": "Maximum retry attempts",
+                "order": 4,
+                "required": False,
                 "type": "int",
+                "value": DEFAULT_MAX_RETRY_ATTEMPS,
             },
             "page_size": {
-                "value": DEFAULT_PAGE_SIZE,
+                "default_value": DEFAULT_PAGE_SIZE,
+                "display": "numeric",
                 "label": "Maximum size of page",
+                "order": 5,
+                "required": False,
                 "type": "int",
+                "value": DEFAULT_PAGE_SIZE,
             },
             "enable_content_extraction": {
-                "value": DEFAULT_CONTENT_EXTRACTION,
```

Comment:
Should we be explicit about the value being either true or false? Just thinking, that it could be a bit better to have it explicitly set.

Reply:
I mean, if we should be explicit about what's the initial value of the toggle.

Reply:
Oh, I think I understand what happened now.
We are explicit about the initial value! I just moved it lower in the dict because I wanted the keys alphabetised 😄

Reply:
L284